### PR TITLE
Feature/stats

### PR DESCRIPTION
**Pull request by AhmedMahdy14**

### CODE REVIEW

File: tests/test_result.py [hunk 1]
```diff
@@ -1,27 +1,71 @@
...
-    def test_basic(self, monkeypatch):
+    def test_basic(self, client, db_session, monkeypatch):
         monkeypatch.setenv("IN_KIND", "FILE")
+        monkeypatch.setenv("MONITORING", "FALSE")
         monkeypatch.setenv("IN_FILENAME", "tests/fixtures/result_input.txt")
         from apihub.result import ResultWriter
 
         try:
             writer = ResultWriter()
             writer.parse_args()
```
Comment (yifan) [inline tests/test_result.py]: There has been a default value change for arguments, we need to use this:
```
            writer.parse_args("--no-in-content-only")
```

Comment (AhmedMahdy14) [reply tests/test_result.py]: ok

Comment (yifan) [reply tests/test_result.py]: @AhmedMahdy14  my mistake, you need to write arguments in an array, `["--no-in-content-only"]`

**Comment by yifan** [comment]
@AhmedMahdy14 I made few changes, please review

**Review by AhmedMahdy14 — COMMENTED** [review]
I reviewed it, don't have comments to change.

**Pull request merged**